docs/source/usersguide/beginners.rst

```diff
@@ -60,7 +60,7 @@ Now let's look at the pros and cons of Monte Carlo methods:
 
 - **Pro**: No mesh generation is required to build geometry. By using
   `constructive solid geometry`_, it's possible to build arbitrarily complex
-  models with curved surfaces.
+  reactor models with curved surfaces.
```

Comment:
I was trying to clarify that for reactor geometries CSG is almost always sufficient, but that other applications might need something more (e.g. CAD).

Rather than specify "reactor" here, which does seem kinda random reading it now, I'd probably remove "arbitrarily" from this sentence instead.